src/AcceptanceTests/ConfigureEndpointAzureServiceBusTransport.cs

```diff
@@ -82,6 +83,8 @@ public Task Configure(string endpointName, EndpointConfiguration configuration,
             endpointOrientedTopology.RegisterPublisher(typeof(When_publishing_from_sendonly.MyEvent), TesingConventions.Conventions.EndpointNamingConvention(typeof(When_publishing_from_sendonly.SendOnlyPublisher)));
             endpointOrientedTopology.RegisterPublisher(typeof(When_subscribing_to_a_base_event.IBaseEvent), TesingConventions.Conventions.EndpointNamingConvention(typeof(When_subscribing_to_a_base_event.Publisher)));
             endpointOrientedTopology.RegisterPublisher(typeof(When_subscribing_to_a_derived_event.SpecificEvent), TesingConventions.Conventions.EndpointNamingConvention(typeof(When_subscribing_to_a_derived_event.Publisher)));
+
+            endpointOrientedTopology.RegisterPublisher(typeof(When_publishing_with_overridden_local_address.MyEvent), TesingConventions.Conventions.EndpointNamingConvention(typeof(When_publishing_with_overridden_local_address.Publisher)));
```

Comment:
So nothing we can or should address in the core?

Reply:
Not really. Core has changed it's API back in v6 and sealed publishers info. Therefore whenever a pub/sub ATT is added to the suite, we have to register publisher for the EndpointOriented topology.

Reply:
Can't remember why some of them don't use the metadata https://github.com/Particular/NServiceBus.AzureServiceBus/blob/8f9aadf9f5f771734b53c840a35f906c948881e9/src/AcceptanceTests/ConfigureEndpointAzureServiceBusTransport.cs#L48